### PR TITLE
Final TS tidyup

### PR DESCRIPTION
**Pull request by anurse**
Just some last TSLint passes. Also I did the "manual remove-and-sort" process on the imports (i.e.: remove them all and Ctrl-. until the errors stop ;))

### CODE REVIEW

File: clients/ts/signalr/spec/HubConnection.spec.ts [hunk 1]
```diff
@@ -2,10 +2,11 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 import { HubConnection } from "../src/HubConnection";
+
```
Comment (mikaelm12) [inline clients/ts/signalr/spec/HubConnection.spec.ts]: You have some new lines in your import statements. Are these intentional/organizational?

Comment (anurse) [reply clients/ts/signalr/spec/HubConnection.spec.ts]: Not this one 😅 

But yes, I tried to separate: External Packages (`buffer`, etc.), Internal **Package** references (`@aspnet/signalr`, etc.), References to product code (`../src/Module`) and finally "Local" references (`./Module`). It seemed like a good way to break things up.

File: clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-import { CompletionMessage, HubMessage, IHubProtocol, ILogger, InvocationMessage, LogLevel, MessageHeaders, MessageType, NullLogger, StreamInvocationMessage, StreamItemMessage, TransferFormat } from "@aspnet/signalr";
 import { Buffer } from "buffer";
 import * as msgpack5 from "msgpack5";
+
+import { CompletionMessage, HubMessage, IHubProtocol, ILogger, InvocationMessage, LogLevel, MessageHeaders, MessageType, NullLogger, StreamInvocationMessage, StreamItemMessage, TransferFormat } from "@aspnet/signalr";
+
 import { BinaryMessageFormat } from "./BinaryMessageFormat";
```
Comment (anurse) [inline clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts]: Here's an example of my intended groupings.

* `buffer` and `msgpack5` are external libraries we use
* `@aspnet/signalr` is the core library, which the MessagePack library references
* `BinaryMessageFormat` is a local module

**Comment by anurse** [comment]
Yeah, that's an upcoming `dev` PR. There are some places that aren't running the linter (unit tests mostly)

**Pull request merged**